spec/xrootd-cmstfc.spec

```diff
@@ -11,10 +10,26 @@ URL: https://github.com/bbockelm/xrootd-cmstfc
 # git-archive master | gzip -7 > ~/rpmbuild/SOURCES/xrootd-lcmaps.tar.gz
 Source0: %{name}.tar.gz
 BuildRoot: %(mktemp -ud %{_tmppath}/%{name}-%{version}-%{release}-XXXXXX)
-BuildRequires: xrootd-libs-devel xerces-c-devel pcre-devel
+
+%define xrootd_current_major 4
+%define xrootd_current_minor 12
+%define xrootd_next_major 5
+
+BuildRequires: xrootd-devel >= 1:%{xrootd_current_major}
+BuildRequires: xrootd-devel <  1:%{xrootd_next_major}
+BuildRequires: pcre-devel
+BuildRequires: xerces-c-devel
 BuildRequires: cmake
+
 Requires: /usr/bin/xrootd pcre xerces-c
 
+#%if 0%{?rhel} < 7
+#Requires: xrootd4 >= 1:4.1.0
+#%else
```

Comment:
Can you remove the debug statements from the spec file?  No obvious reason why we need to keep these.

Reply:
done